pkgs/applications/networking/whoogle-search/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, python3
+, nixosTests
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "whoogle-search";
+  version = "0.7.4";
+
+  src = python3.pkgs.fetchPypi {
+    inherit pname version;
+    hash = "sha256-YaW05CIuwvfzREBKKjTenpSxqz3X/QuDlyenLslTihI=";
+  };
+
+  postInstall = ''
+    mv $out/${python3.sitePackages}/app/static $out/${python3.sitePackages}/app/static_runtime
+
+    # FIXME: Is it possible to specify dataDir from service configuration here?
+    ln -s /var/lib/whoogle-search/static $out/${python3.sitePackages}/app/static
```

Comment:
```suggestion
```
"The install phase is responsible for installing the package in the Nix store under $out" (nixpkgs manual).
The package itself should not depend on a particular data or configuration directory.